VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-global-styles.scss

```diff
@@ -28,14 +28,14 @@
   left: 0;
   bottom: 0;
   right: 0;
-  overflow: auto;
   z-index: 11;
 }
 
 .header-bar {
   display: flex;
   box-shadow: 0px 5px 10px 0px rgb(0 0 0 / 70%);
   box-shadow: 0px 20px 30px 0px rgb(0 0 0 / 30%);
+  overflow-x: auto;
```

Comment:
This should be removed when full responsiveness is implemented for the header

Reply:
is that documented somewhere that we should be removing this?

Reply:
I have added a comment to https://tools.hmcts.net/jira/browse/VIH-8112